spec/index.bs

```diff
@@ -365,32 +378,41 @@ of CPU execution of Javascript.
 
 ## Privacy ## {#security-privacy}
 
+Table-stakes for WebGPU are matching the privacy guarantees of WebGL, but we are looking
+for opportunities where we can futher raise the bar on privacy.
+While these new APIs are powerful and valuable, our users' privacy is essential.
+We're working with our fellow standards body participants to ensure that
+WebGPU is not a vector for tracking users.
+
 ### Machine-specific limits ### {#security-machine-limits}
 
 WebGPU can expose a lot of detail on the underlying GPU architecture and the device geometry.
 This includes available physical adapters, many limits on the GPU and CPU resources
 that could be used (such as the maximum texture size), and any optional hardware-specific
 features or extensions that are available.
 
-WebGPU is designed to have a strong baseline for the limits. This makes it viable (unlike WebGL)
-to *only* expose the baseline limits, trivially reducing the fingerprinting surface.
-The implementations may decide to only expose the baseline even if the hardware is more capable,
-or to support a limited number of higher-capable feature sets.
+This specification is designed with respect to a [privacy budget](https://www.johndcook.com/blog/2019/10/14/privacy-budget/).
```

Comment:
This might be a more appropriate link: https://github.com/bslassey/privacy-budget